pkg/local_object_storage/blobstor/fstree/fstree_write_linux.go

```diff
@@ -243,3 +243,42 @@ func (w *linuxWriter) writeFile(p string, data []byte) error {
 	}
 	return nil
 }
+
+func (w *linuxWriter) writeBatch(objs []putPrm) error {
+	w.batchLock.Lock()
+	defer w.batchLock.Unlock()
+
+	if w.batch != nil {
+		w.batch.sync()
+		err := w.batch.wait()
+		if err != nil {
+			return fmt.Errorf("finalize existing batch: %w", err)
+		}
+		w.batch = nil
+	}
+
+	sb, err := w.newSyncBatch()
+	if err != nil {
+		return fmt.Errorf("create sync batch: %w", err)
+	}
+	defer sb.lock.Unlock()
+	w.batch = sb
+	_ = sb.timer.Stop()
```

Comment:
You can split `newSyncBatch()` to have some time-less batch without creating/stopping it.